projected-resource-requirements.md

```diff
@@ -4,6 +4,31 @@ This document is intended to examine the expected storage, processing, and bandw
 
 ## Basics
 
+### Message Sizes
+
+Here we establish the overhead for various messages in the protocol.
+
+#### Base protocol message overhead
+
+We expect most messages to use the regular MESSAGE type packet from discovery v5 which has 39 bytes of overhead.
```

Comment:
I think the 39 bytes should be 71 bytes. Forgot the source id of 32 bytes perhaps?